locksmith/scripts/deploy-elasticbeanstalk-docker.sh

```diff
@@ -38,10 +39,10 @@ function deploy()
     environment_name=$1
 
     if eb status ${environment_name}; then
-        eb setenv DB_USERNAME=${db_username} DB_PASSWORD=${db_password} DB_NAME=${db_name} DB_HOSTNAME=${db_hostname} NODE_ENV=${node_env} STRIPE_SECRET=${stripe_secret} PURCHASER_CREDENTIALS=${purchaser_credentials} WEB3_PROVIDER_HOST=${web3_provider_host}
+        eb setenv DB_USERNAME=${db_username} DB_PASSWORD=${db_password} DB_NAME=${db_name} DB_HOSTNAME=${db_hostname} NODE_ENV=${node_env} STRIPE_SECRET=${stripe_secret} PURCHASER_CREDENTIALS=${purchaser_credentials} WEB3_PROVIDER_HOST=${web3_provider_host} UNLOCK_CONTRACT_ADDRESS=${unlock_contract_address}
         eb deploy ${environment_name} --label locksmith-${build_id} --message "${message:0:199}"
     else
-        eb create ${environment_name} --envvars DB_USERNAME=${db_username},DB_PASSWORD=${db_password},DB_NAME=${db_name},DB_HOSTNAME=${db_hostname},NODE_ENV=${node_env},STRIPE_SECRET=${stripe_secret},PURCHASER_CREDENTIALS=${purchaser_credentials},WEB3_PROVIDER_HOST=${web3_provider_host} --elb-type classic
+        eb create ${environment_name} --envvars DB_USERNAME=${db_username},DB_PASSWORD=${db_password},DB_NAME=${db_name},DB_HOSTNAME=${db_hostname},NODE_ENV=${node_env},STRIPE_SECRET=${stripe_secret},PURCHASER_CREDENTIALS=${purchaser_credentials},WEB3_PROVIDER_HOST=${web3_provider_host},UNLOCK_CONTRACT_ADDRESS=${unlock_contract_address} --elb-type classic
```

Comment:
Does this need to be `$LOCKSMITH_STAGING_UNLOCK_CONTRACT_ADDRESS`?

Reply:
In this case it does not, the deployment script isn't aware of a staging vs production difference at this point all environment variables are the same and it will use whatever is passed